pages/sign-up/signin.html

```diff
@@ -23,14 +23,18 @@
         <!-- header-starts -->
         <header class="header flex-vCenter">
             <div class="logo-head flex-vCenter">
-                <a href="/index.html"><img class="logo-img" src="/assets/drone-logo-new-removebg-preview.png" alt="drone-logo"></a>
+                <a href="/index.html"><img class="logo-img" src="/assets/drone-logo-new-removebg-preview.png"
```

Comment:
you can add link instead of the path for the image later it helped me when used the same code elsewhere

Reply:
okeiy thanks! will update it later for sure.I remember i was having some problem in this,don't remember though,will change it.